app/Services/ModuleService.php

```diff
@@ -70,4 +78,101 @@ public function getAdminLinks(): array
     {
         return self::$adminLinks;
     }
+
+    public function createModule($array): bool
+    {
+        $orig_file = $array['file'];
+        $file_ext = $orig_file->getClientOriginalExtension();
+        $allowed_extensions = ['zip', 'tar', 'gz'];
+
+        if (!in_array($file_ext, $allowed_extensions)) {
+            return false;
+        }
+
+        $module = null;
+        $temp = storage_path('app/temp_modules');
```

Comment:
Change this to `app/tmp`, and then create a `tmp` folder with a `.gitignore` file in there, you can just copy the current `app/import` folder. Just have to make sure that this is there